config/views/static.py

```diff
@@ -146,6 +159,10 @@ class TermsOfUseView(TemplateViewWithContext):
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         context["feedback_survey_type"] = "support"
+        context["breadcrumbs"] = [
+            {"url": "/terms-and-policies", "text": "Terms and policies"},
+            {"text": "Terms of Use"},
```

Comment:
Should this be lowercase "use" to match everything else being sentence case?

Reply:
@jacksonj04 thanks for spotting this will update